### PR TITLE
Pages Created CSV export

### PR DESCRIPTION
**Pull request by MusikAnimal**
Minor refactoring in EventDataController

Bug: T206058

### CODE REVIEW

File: src/AppBundle/Controller/EventDataController.php [hunk 1]
```diff
@@ -136,6 +127,15 @@ private function getFormattedResponse(string $format, string $template, array $p
         $contentType = $formatMap[$format] ?? 'text/html';
         $response->headers->set('Content-Type', $contentType);
 
+        // Prettier file names for CSV.
+        $eventName = $this->getFilenameFriendlyEventName();
```
Comment (samwilson) [inline src/AppBundle/Controller/EventDataController.php]: I know it's not being introduced in this patch, but if an event name contains double quotes then it gets truncated to the first of em.

Comment (samwilson) [reply src/AppBundle/Controller/EventDataController.php]: (I meant to make this comment on the header-setting string construction below, actually.)

Comment (MusikAnimal) [reply src/AppBundle/Controller/EventDataController.php]: Hmm, I don't quite follow. What gets truncated? An event with the title `"Foo bar"` produces `event_summary__Foo_bar_.csv`. The double underscores aren't ideal I guess but I don't think it's a problem.

**Comment by MusikAnimal** [comment]
I'll go ahead and merge with Sam's approval. If we uncover an issue with the file names we can take care of that with a different PR.

File: src/AppBundle/Controller/EventDataController.php (hunk 1 above)
Comment (samwilson) [reply src/AppBundle/Controller/EventDataController.php]: Hm, odd. I have an event called `Test Event "One"` and the header that's sent with the CSV is `attachment; filename="pages_created_Test_Event_"One".csv"` and I'm prompted to save a file called `pages_created_Test_Event_`... maybe this is something to do with different browser's handling of the wayward quotation marks?

Comment (MusikAnimal) [reply src/AppBundle/Controller/EventDataController.php]: That is weird! And that should definitely be fixed. I've created https://github.com/wikimedia/eventmetrics/pull/205 but I can't repro so not sure if that does it.

**Pull request merged**